tiatoolbox/models/architecture/vanilla.py

```diff
@@ -193,7 +200,8 @@ class CNNBackbone(ModelABC):
 
     """
 
-    def __init__(self, backbone):
+    def __init__(self, backbone) -> None:
+        """Initializes :class:`CNNBackbone`."""
```

Comment:
```suggestion
        """Initialize :class:`CNNBackbone`."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/c2e1cd4ccfd6aaeb32664f23d4f6634cddb90090